src/App.jsx

```diff
@@ -1,3 +1,12 @@
+import { NewTodoForm } from "./components/NewTodoForm";
```

Comment:
Very impressed with your todo app. The text that tells the user what to type in the input field, the check all tasks-button, was great ideas i wish i've thought about too:)  nice, tidy, and simple design too.